nextflow.config

```diff
@@ -3,95 +3,105 @@
+  igenomes_base = "./iGenomes"
+  tracedir = "${params.outdir}/pipeline_info"
+  awsqueue = false
+  awsregion = 'eu-west-1'
+  igenomesIgnore = false
+  custom_config_version = 'master'
+  custom_config_base = "https://raw.githubusercontent.com/nf-core/configs/${params.custom_config_version}"
+  hostnames = false
+  config_profile_description = false
+  config_profile_contact = false
+  config_profile_url = false
+}
 
-  // Custom trimming options
-  length = 18
-  clip_R1 = 0
-  three_prime_clip_R1 = 0
-  protocol = "illumina"
-  three_prime_adapter = ""
+// Container slug. Stable releases should specify release tag!
+// Developmental code should specify :dev
+process.container = 'nfcore/smrnaseq:dev'
+
+// Load base.config by default for all pipelines
+includeConfig 'conf/base.config'
+
+// Load nf-core custom profiles from different Institutions
+try {
+  includeConfig "${params.custom_config_base}/nfcore_custom.config"
+} catch (Exception e) {
+  System.err.println("WARNING: Could not load nf-core/config profiles: ${params.custom_config_base}/nfcore_custom.config")
 }
 
 profiles {
-
-  standard {
-    includeConfig 'conf/base.config'
-    includeConfig 'conf/igenomes.config'
-  }
+  awsbatch { includeConfig 'conf/awsbatch.config' }
   conda { process.conda = "$baseDir/environment.yml" }
+  debug { process.beforeScript = 'echo $HOSTNAME' }
   docker { docker.enabled = true }
   singularity { singularity.enabled = true }
-  uppmax {
-    includeConfig 'conf/base.config'
-    includeConfig 'conf/uppmax.config'
-    includeConfig 'conf/igenomes.config'
-  }
-  uppmax_modules {
-    includeConfig 'conf/base.config'
-    includeConfig 'conf/uppmax-modules.config'
-    includeConfig 'conf/igenomes.config'
-  }
-  test {
-    includeConfig 'conf/base.config'
-    includeConfig 'conf/test.config'
-  }
-  none {
-    // Don't load any config (for use with custom home configs)
-  }
-
+  test { includeConfig 'conf/test.config' }
 }
 
+// Load igenomes.config if required
+if(!params.igenomesIgnore){
+  includeConfig 'conf/igenomes.config'
+}
 
 // Capture exit codes from upstream processes when piping
 process.shell = ['/bin/bash', '-euo', 'pipefail']
 
 timeline {
   enabled = true
-  file = "${params.outdir}/pipeline_info/smRNAseq_timeline.html"
+  file = "${params.tracedir}/execution_timeline.html"
 }
 report {
   enabled = true
-  file = "${params.outdir}/pipeline_info/smRNAseq_report.html"
+  file = "${params.tracedir}/execution_report.html"
 }
 trace {
   enabled = true
-  file = "${params.outdir}/pipeline_info/smRNAseq_trace.txt"
+  file = "${params.tracedir}/execution_trace.txt"
 }
 dag {
   enabled = true
-  file = "${params.outdir}/pipeline_info/smRNAseq_dag.svg"
+  file = "${params.tracedir}/pipeline_dag.svg"
 }
 
 manifest {
+  name = 'nf-core/smrnaseq'
+  author = 'Phil Ewels <phil.ewels@scilifelab.se>, Chuan Wang <chuan.wang@scilifelab.se>, Rickard Hammarén <rickard.hammaren@scilifelab.se>, Lorena Pantano <lorena.pantano@gmail.com>'
```

Comment:
You may have to update the `author` and `description` lines in the `TEMPLATE` branch you created too.

Reply:
As far as I understand, the `TEMPLATE` branch is just a vanilla version of `tools` for your pipeline. You should be able to clone it and edit these two fields so they match the updated versions. Then you should be able to create a PR to `nf-core/smrnaseq` to add the `TEMPLATE` branch here.  Is that right @apeltzer  ?

Reply:
I'll take that into account when I do the PR for the TEMPLATE branch.